bin/sl-run.js

```diff
@@ -17,6 +17,14 @@ var agentOptions = {
   logger: config.logger,
 };
 
+var tracer = require('concurix');
```

Comment:
should be optional, probably controlled by same --profile/--no-profile flag that controls agent